src/api/app/models/bs_request.rb

```diff
@@ -65,6 +65,7 @@ class BsRequest < ApplicationRecord
     where(state: 'review', id: Review.where(review_attributes).where(state: 'new').select(:bs_request_id))
       .includes(:reviews)
   }
+  scope :with_state, ->(state) { where(state: state) }
```

Comment:
Congratulations, you just reinvented `.where`.... 😜

Reply:
Too much copy/paste/replace automation made dumb code. Thanks for raising this up, this and the other comment. I'm going to squeeze it

Reply:
Done